ui/src/main/webapp/src/app/project/migration-project.service.ts

```diff
@@ -132,6 +139,20 @@ export class MigrationProjectService extends AbstractService {
             .catch(this.handleError);
     }
 
+    @Cached('project', {minutes: 1})
+    getAllIncludingProvisional(): Observable<Array<MigrationProject & HasAppCount>> {
+        return this._http.get(Constants.REST_BASE + this.GET_MIGRATION_PROJECTS_URL + "?includeProvisional=true")
```

Comment:
I'm ok taking that risk. I think we should only return non-provisional items in this query.